### PR TITLE
Pset updater add input and output

### LINKED ISSUE #100: [Pset] Updater add input and output
The updater should let a user adding a new input to the partial transaction. On the contrary, at the moment it just lets to add info to the partial inputs for existing transaction inputs.

We should add a new method `AddInput` to the `*Updater` type that expects a `*transaction.TxInput`, calls the `AddInput` function of the underlying `Pset.UnsignedTx` field and increases the length of the `Pset.Inputs` array.

The same thing should be done for being able to also add outputs to the partial transaction.

### PR DESCRIPTION
**Pull request by sekulicd**
Before this PSET Updater was not able to add inputs or outputs to underlaying unsigned transaction.

After the methods AddInput and AddOutput are added to PSET Updater role user can add input and output transaction to the underlying unsigned transaction.

It closes #100 .

Please @altafan can you review this?

### CODE REVIEW

File: pset/updater_test.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	"encoding/hex"
+	"github.com/stretchr/testify/assert"
+	"github.com/vulpemventures/go-elements/transaction"
+	"testing"
+)
+
+func TestUpdaterAddInput(t *testing.T) {
+	inputs := make([]*transaction.TxInput, 0)
+	output := make([]*transaction.TxOutput, 0)
+	pset, err := New(inputs, output, 2, 0)
```
Comment (altafan) [inline pset/updater_test.go:13]: please, call this `p`, since pset is the name of the package.

File: pset/updater_test.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import (
+	"encoding/hex"
+	"github.com/stretchr/testify/assert"
+	"github.com/vulpemventures/go-elements/transaction"
+	"testing"
+)
+
+func TestUpdaterAddInput(t *testing.T) {
+	inputs := make([]*transaction.TxInput, 0)
+	output := make([]*transaction.TxOutput, 0)
```
Comment (altafan) [inline pset/updater_test.go:12]: should be `outputs`

File: pset/updater_test.go (hunk 1 above)
Comment (sekulicd) [reply pset/updater_test.go:13]: Done

File: pset/updater_test.go (hunk 2 above)
Comment (sekulicd) [reply pset/updater_test.go:12]: Done

File: pset/updater_test.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		"000102030405060708090a0b0c0d0e0f000102030405060708090a0b0c0d0e0f")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	txInput := transaction.TxInput{
+		Hash:  hash,
+		Index: 2,
+	}
+	updater.AddInput(&txInput)
```
Comment (altafan) [inline pset/updater_test.go:32]: We should assert that, before this function is called, the transaction has 0 inputs and 0 outputs

Comment (sekulicd) [reply pset/updater_test.go:36]: Done

**Comment by altafan** [comment]
Please sync this branch with master, since it's behind 1 commit

**Comment by sekulicd** [comment]
> Please sync this branch with master, since it's behind 1 commit.
> @sekulicd

Done

**Pull request merged**